tests/ci/run_build.py

```diff
@@ -237,9 +237,17 @@ def build_srpm_and_rpm(
 failed = []
 rebuild_success = []
 total = 0
+docs_spec_executed = False
 
 for spec in args.specfiles:
-    if not spec.endswith('.spec'):
+    if 'components/admin/docs/SPECS/docs.spec' == spec:
+        if docs_spec_executed:
+            continue
+        docs_spec_executed = True
+    elif not docs_spec_executed and 'docs/recipes/install/' in spec:
```

Comment:
Should I add logic for the get_source.sh plugin too ?
```suggestion
    elif not docs_spec_executed and ('docs/recipes/install/' in spec or 'components/admin/docs/SOURCES/` in spec):
```